### PR TITLE
Added Kanban Board

### LINKED ISSUE #716: Adding a Kanban Board
**Project title**
Kanban Board

**Project Description**
A kanban board is one of the tools that can be used to implement kanban to manage work at a personal or organizational level. Kanban boards visually depict work at various stages of a process using cards to represent work items and columns to represent each stage of the process.

**Stack**:  
And in order to tick the check box just but x inside them for example - [x] like this. Please delete options that are not relevant.

- [x] Html
- [x] CSS
- [x] JavaScript

### PR DESCRIPTION
**Pull request by LiQuiD-404**
<hr>

## Description 📜

A simple Javascript Organizer App.

Fixes #716

<hr>

## Type of change 📝

- [x] Add a new project

<hr>

## Domain of Contribution 📊

- [x] Project using HTML, CSS & JS

<hr>
 
## Checklist ✅

- [x] I have performed a self-review of my own code or work.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I'm GSSoC'22 contributor

<hr>
 
## Screenshots / Gif (Recommended) 📸
snip1

### CODE REVIEW

File: Kanban Board/Readme.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  
+  
+  
+  
+  
+  
+
+
+  
+
```
Comment (Mrjoy832) [inline Kanban Board/Readme.md:188]: Add Demo Video

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

**Comment by LiQuiD-404** [comment]
@Mrjoy832 Added a Demo  Video. Kindly review

**Comment by LiQuiD-404** [comment]
> @Mrjoy832 Added a Demo Video. Kindly review



> If UI is somehow more better I would give you Label-3

No issues. Next time i will try to do better

**Comment by Mrjoy832** [comment]
> If UI is somehow more better I would give you Label-3

Checked Readme video...Nice Work

**Pull request merged**